content/docs/started/k8s/overview.md

```diff
@@ -4,7 +4,13 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
 weight = 1
 +++
 
-Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
+Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster. The cluster must meet the following requirements:
+
+  * Kubernetes version {{% kubernetes-min-version %}} or later.
+  * A minimum of 0.6 CPU in cluster. (Reserved for 3 replicated ambassador pods
+    and according to your needs add additional CPUs.)
+  * Node with storage >= 10 GB. (Due to the ML libraries and third party packages
+    bundled in the Kubeflow Docker images.)
```

Comment:
The requirement for only 0.6 CPU seems outdated, my experience is that Kubeflow is much more resource intensive.
In addition, ambassador has been replaced by Istio so we should probably remove the parenthesis referring to it.

Reply:
I've cleaned up this section to better match the information on minikube, minikf and multipass.